feat(sig-services): add contract-duration requirement (#103)
open-services-group/community: sig-services/service-requirements/contract-duration.md

---
title: contract duration
persona: service consumer, service provider/owner
phase: service-operational
---

> This requirement states that a service should have a defined contract duration. Contract duration states the start and the end date of the subscription to the service or that the subscription doesn't have an end date.

Reviewer: ```suggestion
This requirement states that a service should have a defined contract duration. Contract duration states the start and the end of the subscription to the service or the conditions under which a subscription is terminated.
```

> If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers acknowledgement.

Reviewer: ```suggestion
If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers explicit acknowledgement.
```

> It should also state under what circumstances can the service provider involuntarily unsubscribe the consumer (e. g., breach of Terms & Conditions) from the service. Similarly the service consumer should also be able end the service subscription before the end date (if there is one) on his own just by unsubscribing from the service.

Reviewer: ```suggestion
It should also state under what circumstances the service provider can involuntarily terminate the consumer's subscription (e. g., breach of Terms & Conditions). Additionally the service consumer should be advised how to end the service subscription on his own just at any time, especially in a case when the contract specifies a distinct end date of a subscription and what terms apply in such scenario.
```

## Requirements

- Create a file/page which defines the contract duration
- Define cases when the consumer will be involuntarily unsubscribed from the service
- Define that the consumer can unsubscribe from the service on his own at any time
- Define whether the service subscription can be extended and whether it auto extends

> 

Reviewer: ```suggestion
- Contract duration is not a simple link to lengthy Terms and Conditions document but rather a simple list that links back to the Terms and Conditions
- Contract duration is not a marketing or layer material. That means it is worded accordingly to the target audience - users. It is understandable to the user.

```

## Reasoning

> Defining a contract duration brings transparency to booth the consumer and the provider and creates a clear understanding of what the subscription duration is and rules which the consumer should follow to stay subscribed to the service.

Reviewer: ```suggestion
Defining a contract duration brings transparency to booth the consumer and the provider and creates a clear understanding of what the subscription duration is and rules which the consumer should follow to stay subscribed to the service as well as how to leave the service.
```

phase: service-operational
---

This requirement states that a service should have a defined contract duration. Contract duration states a start and an end of a subscription to the service and the conditions under which the subscription is terminated.

If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers explicit acknowledgement.

It states the circumstances when the service provider can involuntarily terminate the consumer's subscription. Additionally the service consumer should be advised how to end the service subscription on his own just at any time, especially in a case when the contract specifies a distinct end date of the subscription and what terms apply in such scenario.

## Requirements

- Create a file/page which defines the contract duration
- Define cases when the consumer will be involuntarily unsubscribed from the service
- Define if and how the consumer can unsubscribe from the service on his own at any time
- Define whether the service subscription can be extended and whether it auto extends

> - Contract duration is not a marketing or lawyer material. That means it is worded accordingly to the target audience - users. It should be easy to read and understandable.

Reviewer: ```suggestion
- Contract duration is not a marketing or lawyer material. That means it is worded accordingly to the target audience - users. It should be easy to read and understand.
```

"... to read and understandable" sounds a bit odd.

If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers explicit acknowledgement.

It should also state under what circumstances the service provider can involuntarily terminate the consumer's subscription (e. g., breach of Terms and Conditions). Additionally the service consumer should be advised how to end the service subscription on his own just at any time, especially in a case when the contract specifies a distinct end date of a subscription and what terms apply in such scenario.

## Requirements

- Create a file/page which defines the contract duration
- Define cases when the consumer will be involuntarily unsubscribed from the service
- Define that the consumer can unsubscribe from the service on his own at any time
- Define whether the service subscription can be extended and whether it auto extends

> - Contract duration is not a simple link to lengthy Terms and Conditions document but rather a simple list that links back to the Terms and Conditions

Reviewer 2: What is this point trying to convey? Would there be a separate Terms and Conditions document that the contract document should refer to?

Author: Resolved in a meeting.

> - Contract duration is not a marketing or layer material. That means it is worded accordingly to the target audience -- users. It is understandable to the user.

Reviewer 2: ```suggestion
- Contract duration is not a marketing or layer material. That means it is worded accordingly to the target audience -- users. It should be understandable to the users.
```

## Reasoning

> Defining a contract duration brings transparency to booth the consumer and the provider and creates a clear understanding of what the subscription duration is and rules which the consumer should follow to stay subscribed to the service as well as how to leave the service.

Reviewer 2: ```suggestion
Defining a contract duration manages expectations of both the service consumer and the provider transparently. It specifies the subscription duration, and the rules that the consumer should follow to stay subscribed and to leave the service.
```

Author: ```suggestion
Defining a contract duration manages expectations of both the service consumer and the provider transparently. It specifies the subscription duration, and the rules that the consumer should follow to stay subscribed and how to leave the service.
```
